tools/tplgtool2.py

```diff
@@ -729,13 +729,16 @@ def print_pcm_info(self):
             fmt_list = self.get_pcm_fmt(pcm)
             cap_index = int(pcm_type == "capture") # 1 for capture, 0 for playback
             cap = pcm["caps"][cap_index]
-            fmt = fmt_list[cap_index][0] # only show first format
+            fmts = ''
+            for fmt in fmt_list[cap_index]:
+                fmts = fmts + fmt + ' '
             rate_min = cap["rate_min"]
             rate_max = cap["rate_max"]
             ch_min = cap["channels_min"]
             ch_max = cap["channels_max"]
-            print(f"pcm={name};id={pcm_id};type={pcm_type};fmt={fmt};"
-            f"rate_min={rate_min};rate_max={rate_max};ch_min={ch_min};ch_max={ch_max};")
+            print(f"pcm {pcm_id:>2}: {name:<22};type={pcm_type:<8};"
```

Comment:
The output looks much better. However the previous code was producing consistent output easy to parse because everything was formated `key=value;`. this PR breaks that.

I think we can have both the parsable output AND the indentation; try something like this:

```python
name_semi = pcm["pcm_name"] + ";"
.
.
.

print(f"pcm={pcm_id:>2}; name={name_semi:<21} ...")
```

Reply:
let's find the longest filed of each column, and use it for formatting.

Reply:
@marc-hb @aiChaoSONG Thanks for your review and suggestions!  All comments addressed in v2.

Reply:
@marc-hb Chao confirmed our test cases use output of ./tools/sof-tplgreader.py to check PCM streams, not output of this tplgtool2.py So we can make the output more human friendly. Now I still follow `key=value;` in v2 patches.

Reply:
> Chao confirmed our test cases use output of ./tools/sof-tplgreader.py to check PCM streams, not output of this tplgtool2.py

We don't know who parses the output of this script. It has existed for a very long time.

> Now I still follow key=value   ;

Thanks! The new whitespace could still break someone but that should be very quick and easy to fix.